### PR TITLE
add database poppulator

### PR DESCRIPTION
**Pull request by grechman**
please, take a look

### CODE REVIEW

File: pocket_world_application/src/main/java/org/exschool/pocketworld/init/Poppulater.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    @Autowired
+    private CityService cityService;
+
+    @Autowired
+    private ResourceBuildingService resourceBuildingService;
+
+
+    @PostConstruct
+    void fillDatabase(){
+        Long player1Id = createPlayer("login-1", new PlayerResources(10, 15, 18, 20));
```
Comment (cat-eat-dog) [inline pocket_world_application/src/main/java/org/exschool/pocketworld/init/Poppulater.java]: database should be filled with data only if database is empty

**Comment by grechman** [comment]
Fix according to comments

File: pocket_world_application/src/main/java/org/exschool/pocketworld/init/Populator.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    }
+
+    private Long getPlayerId(String playerLogin) {
+        return playerService.getPlayerByLogin(playerLogin).getId();
+    }
+
+    private boolean isPlayerExist(String playerLogin) {
+        return playerService.getPlayerByLogin(playerLogin) != null;
+    }
+
```
Comment (cat-eat-dog) [inline pocket_world_application/src/main/java/org/exschool/pocketworld/init/Populator.java]: better move methods below to appropriate services

File: pocket_world_application/src/main/java/org/exschool/pocketworld/init/Populator.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            createResourceBuilding(city2Id, ResourceType.TIMBER, 3, 0);
+        }
+    }
+
+    private boolean isResBuildingExist(Long cityId, int position) {
+        return resourceBuildingService.getAtPosition(cityId, position) != null;
+    }
+
+    private Long createResourceBuilding(Long cityId, ResourceType resourceType, int position, int level) {
+        ResourceBuilding resourceBuilding = ResourceBuildingBuilder.builder()
```
Comment (cat-eat-dog) [inline pocket_world_application/src/main/java/org/exschool/pocketworld/init/Populator.java]: make separate method for creating entities

**Pull request merged**